bin/check-nvmrc.js

```diff
@@ -0,0 +1,33 @@
+#! /usr/bin/env node
+
+'use strict'
+
+var fs = require('fs')
+var path = require('path')
+
+fs.readFile(path.join(__dirname, '../.nvmrc'), 'utf8', function (error, data) {
+  if (error) throw error
+  var expectedVersion = data.trim()
+  var currentVersion = process.version.replace('v', '')
+
+  var versionMatchesExactly = expectedVersion === currentVersion
+  var versionMatchesMajor = expectedVersion.substring(1, 0) === currentVersion.substring(1, 0)
+
+  if (versionMatchesExactly) {
+    process.exit()
+  }
+
+  if (versionMatchesMajor) {
+    console.log('' +
+      'Warning: You are using Node.js version ' + currentVersion + ' which we do not use. ' +
+      'You may encounter issues consider installing Node.js version ' + expectedVersion + '.' +
```

Comment:
Think there should be some sort of punctuation between `issues` and `consider`?

Reply:
@amyhupe mind having a look at the language in these messages?

Reply:
I'll update this